src/rsz/test/remove_buffers2.ok

```diff
@@ -4,38 +4,41 @@
 [INFO ODB-0225]     Created 135 library cells
 [INFO ODB-0226] Finished LEF file:  Nangate45/Nangate45.lef
 [INFO ODB-0128] Design: top
-[WARNING ODB-0100] error: netlist component-pin (i1, Z) is not defined
 [INFO ODB-0130]     Created 3 pins.
-[INFO ODB-0131]     Created 3 components and 12 component-terminals.
-[INFO ODB-0132]     Created 2 special nets and 6 connections.
-[INFO ODB-0133]     Created 4 nets and 5 connections.
-Startpoint: b1/A (internal pin)
-Endpoint: out1 (output port)
+[INFO ODB-0131]     Created 2 components and 8 component-terminals.
+[INFO ODB-0132]     Created 2 special nets and 4 connections.
+[INFO ODB-0133]     Created 3 nets and 4 connections.
+Startpoint: in1 (input port)
+Endpoint: out2 (output port)
 Path Group: (none)
 Path Type: max
 
   Delay    Time   Description
 ---------------------------------------------------------
-   0.00    0.00 v b1/A (BUF_X2)
-   0.02    0.02 v b1/Z (BUF_X2)
-   0.00    0.02 v out1 (out)
-           0.02   data arrival time
+   0.00    0.00 v input external delay
+   0.00    0.00 v in1 (in)
+   0.01    0.01 ^ i1/ZN (INV_X1)
+   0.02    0.03 ^ b2/Z (BUF_X4)
+   0.00    0.03 ^ out2 (out)
+           0.03   data arrival time
 ---------------------------------------------------------
 (Path is unconstrained)
 
 
-[INFO RSZ-0026] Removed 1 buffers.
-Startpoint: b2/A (internal pin)
+[INFO RSZ-0026] Removed 0 buffers.
```

Comment:
Doesn't this kind of break the test?

Reply:
The test was already pretty broken with a bad DEF file.  I'll see if there is something better I can do with it.

Reply:
looks like in the original maybe
```
- n1 ( i1 Z ) ( b1 A ) ( b2 A ) ;
```
was supposed to be
```
- n1 ( i1 ZN ) ( b1 A ) ( b2 A ) ;
```

Reply:
You'll see that's the change I made in this PR.  However it changes the result to no buffer removal.

Reply:
I see that you changed that in the `.defok` but I'm talking about the input `.def`. Here is what I see when I make the change:
Input def:
```diff
diff --git a/src/rsz/test/remove_buffers2.def b/src/rsz/test/remove_buffers2.def
index 65b3f7583..e01762140 100644
--- a/src/rsz/test/remove_buffers2.def
+++ b/src/rsz/test/remove_buffers2.def
@@ -30,7 +30,7 @@ END SPECIALNETS
NETS 10 ;
 - in1 ( PIN in1 ) ( i1 A ) ;
- - n1 ( i1 Z ) ( b1 A ) ( b2 A ) ;
+ - n1 ( i1 ZN ) ( b1 A ) ( b2 A ) ;
 - out1 ( PIN out1 ) ( b1 Z )  ;
 - out2 ( PIN out2 ) ( b2 Z )  ;
END NETS
```
Result def:
```diff
22c22
diff --git a/src/rsz/test/remove_buffers2.defok b/src/rsz/test/remove_buffers2.defok
index 583b878e1..e4b9f8fa8 100644
--- a/src/rsz/test/remove_buffers2.defok
+++ b/src/rsz/test/remove_buffers2.defok
@@ -19,7 +19,7 @@ SPECIALNETS 2 ;
 END SPECIALNETS
 NETS 3 ;
     - in1 ( PIN in1 ) ( i1 A ) + USE SIGNAL ;
-    - out1 ( PIN out1 ) ( b2 A ) + USE SIGNAL ;
+    - out1 ( PIN out1 ) ( b2 A ) ( i1 ZN ) + USE SIGNAL ;
     - out2 ( PIN out2 ) ( b2 Z ) + USE SIGNAL ;
 END NETS
 END DESIGN
```
ok file:
```diff
diff --git a/src/rsz/test/remove_buffers2.ok b/src/rsz/test/remove_buffers2.ok
index 13f2e800b..3f28662ae 100644
--- a/src/rsz/test/remove_buffers2.ok
+++ b/src/rsz/test/remove_buffers2.ok
@@ -4,38 +4,41 @@
 [INFO ODB-0225]     Created 135 library cells
 [INFO ODB-0226] Finished LEF file:  Nangate45/Nangate45.lef
 [INFO ODB-0128] Design: top
-[WARNING ODB-0100] error: netlist component-pin (i1, Z) is not defined
 [INFO ODB-0130]     Created 3 pins.
 [INFO ODB-0131]     Created 3 components and 12 component-terminals.
 [INFO ODB-0132]     Created 2 special nets and 6 connections.
-[INFO ODB-0133]     Created 4 nets and 5 connections.
-Startpoint: b1/A (internal pin)
+[INFO ODB-0133]     Created 4 nets and 6 connections.
+Startpoint: in1 (input port)
 Endpoint: out1 (output port)
 Path Group: (none)
 Path Type: max
 
   Delay    Time   Description
 ---------------------------------------------------------
-   0.00    0.00 v b1/A (BUF_X2)
-   0.02    0.02 v b1/Z (BUF_X2)
-   0.00    0.02 v out1 (out)
-           0.02   data arrival time
+   0.00    0.00 v input external delay
+   0.00    0.00 v in1 (in)
+   0.02    0.02 ^ i1/ZN (INV_X1)
+   0.02    0.04 ^ b1/Z (BUF_X2)
+   0.00    0.04 ^ out1 (out)
+           0.04   data arrival time
 ---------------------------------------------------------
 (Path is unconstrained)
 
 
 [INFO RSZ-0026] Removed 1 buffers.
-Startpoint: b2/A (internal pin)
+Startpoint: in1 (input port)
 Endpoint: out2 (output port)
 Path Group: (none)
 Path Type: max
 
   Delay    Time   Description
 ---------------------------------------------------------
-   0.00    0.00 v b2/A (BUF_X4)
-   0.02    0.02 v b2/Z (BUF_X4)
-   0.00    0.02 v out2 (out)
-           0.02   data arrival time
+   0.00    0.00 v input external delay
+   0.00    0.00 v in1 (in)
+   0.01    0.01 ^ i1/ZN (INV_X1)
+   0.02    0.03 ^ b2/Z (BUF_X4)
+   0.00    0.03 ^ out2 (out)
+           0.03   data arrival time
 ---------------------------------------------------------
 (Path is unconstrained)
```

Reply:
I think this test was created by hand and it was a simple oversight that the `INV_X1` ouput pin is `ZN` not `Z`

Reply:
Created #2465 if you want to change it